packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -366,7 +366,9 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
               options={{
                 callControls: false,
                 /* @conditional-compile-remove(call-readiness) */
-                devicePermissions: props.devicePermissions
+                devicePermissions: props.devicePermissions,
+                /* @conditional-compile-remove(call-readiness) */
+                callReadinessOptedIn: true
```

Comment:
Same as my other comment from the CallScreen.tsx
Should this be set to true? Wouldn't the prop be passed down?
The Calling Sample and CallWithChat sample would pass down the prop to set this value

Reply:
fixed